content/en/docs/architecture/bugzilla.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "Bugzilla Integration"
+description: An overview of the integration of Bugzilla with the Openshift CI
+---
+
+Many automated processes exist to help you navigate the OpenShift Bugzilla workflow when you're fixing a Bugzilla bug with a
+GitHub Pull Request. Read up on the overall process [here][6] before diving into the specifics of automation.
+
+## Linking Pull Requests to Bugzilla bugs
+
+In order to declare that your pull request will fix a Bugzilla bug, all you need to do is add the bug number as a prefix to your pull
+request title, like `Bug 123: Rest of the PR title`. A PR can at most reference one Bugzilla bug.
+
+This will make the integration run a couple of validations. It will comment on GitHub with a detailled explanation of the
+validations it ran and their result, so you can easily fix issues if needed. It will also add a label that indicates the success
+of those validations to integrate with the merge automation.
+
+You can find a detailed list of the bugzilla plugin settings for your repo in the [Prow Plugins][5] page by first selecting your
+repository in the top left and then clicking on `details` of the Bugzilla plugin.
```

Comment:
nit: we should note to use `/bugzilla refresh` if Bugzilla state changes and re-evaluation is needed